app/Http/Controllers/CategoryController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Category;
+
+class CategoryController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $categories= Category::all();
```

Comment:
Return resource directly
```suggestion
        $categories= Category::all();
```